doc/zh-cn/plugins/dynamic-upstream.md

```diff
@@ -0,0 +1,290 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+- [English](../../plugins/dynamic-upstream.md)
+
+# 目录
+
+  - [目录](#目录)
+  - [名字](#名字)
+  - [属性](#属性)
+  - [如何启用](#如何启用)
+    - [灰度发布](#蓝绿发布)
+    - [蓝绿发布](#蓝绿发布)
+    - [自定义发布](#自定义发布)
+  - [测试插件](#测试插件)
+    - [灰度测试](#灰度测试)
+    - [蓝绿测试](#蓝绿测试)
+    - [自定义测试](#自定义测试)
+  - [禁用插件](#禁用插件)
+
+
+## 名字
+
+`dynamic-upstream` 对请求进行条件匹配并按比率切分上游。
+
+## 属性
+
+| 参数名        | 类型          | 可选项 | 默认值 | 有效值 | 描述                 |
+| ------------ | ------------- | ------ | ------ | ------ | -------------------- |
+| rules        | array[object] | 必需  |       |        | 插件的配置列表 |
+| match        | array[object] | 必需  |        |        | 匹配规则列表 |
```

Comment:
they are wrong.
we should use `rules[].match` here.

Reply:
Ok, I will do it later.